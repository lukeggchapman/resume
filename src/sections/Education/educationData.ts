import mdToHast from 'utils/mdToHast'

import macquarieData from 'content/education/2015-macquarie.md'
import macquarieLogo from 'content/education/macquarie-uni.png'

const experienceData = [
  {
    frontmatter: macquarieData.data,
    logo: macquarieLogo,
    htmlAst: mdToHast(macquarieData.content),
  },
]

export default experienceData
