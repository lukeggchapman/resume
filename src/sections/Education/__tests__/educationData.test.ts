import educationData from '../educationData'

jest.mock('utils/parseMarkdownToHtmlAst', () => (md: string) => ({
  frontmatter: { frontmatterFor: md },
  htmlAst: `htmlAst(${md})`,
}))
jest.mock('content/education/2015-macquarie.md', () => 'macquarieContentMock', {
  virtual: true,
})
jest.mock('content/education/macquarie-uni.png', () => 'macquarieLogoMock')

describe('educationData', () => {
  it('should have the correct data', () => {
    expect(educationData).toMatchSnapshot()
  })
})
