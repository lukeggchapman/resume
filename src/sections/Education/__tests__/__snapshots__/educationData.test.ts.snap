// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`educationData should have the correct data 1`] = `
Array [
  Object {
    "frontmatter": "macquarieFrontmatterMock",
    "htmlAst": "mdToHast(macquarieContentMock)",
    "logo": "macquarieLogoMock",
  },
]
`;
